DotNET.Fundamentals.sln

```diff
@@ -77,8 +77,6 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Rules", "Source\Rules\Rules
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ResourceTypes", "Source\ResourceTypes\ResourceTypes.csproj", "{6FACBDA5-1C3D-43AE-BEAE-FD39EDC6A4FD}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ResourceTypes", "Specifications\ResourceTypes\ResourceTypes.csproj", "{77270B8D-52DF-4A95-BE22-B2CE234E2657}"

```

Comment:
Why was this removed? Seems unrelated to what we are doing...

Reply:
It is. I was just in a cleanup mood and removed lots of folders on my local system